tests/snowflake/hooks/test_snowflake_async.py

```diff
@@ -90,5 +90,64 @@ async def test_get_query_status(self, mock_conn, query_ids, expected_state, expe
         conn.is_still_running.return_value = False
         conn.get_query_status.return_value = expected_state
         result = await hook.get_query_status(query_ids=query_ids)
-        print("Test case")
         assert result == expected_result
+
```

Comment:
Adding necessary doc strings and comments on the test would be useful in all the tests